lib/daru/vector.rb

```diff
@@ -879,41 +879,86 @@ def to_s
       to_html
     end
 
-    # Create a summary of the Vector using Report Builder.
-    def summary(method=:to_text)
-      ReportBuilder.new(no_title: true).add(self).send(method)
+    # Create a summary of the Vector
+    # @params [Fixnum] indent_level
+    # @return [String] String containing the summary of the Vector
+    # @example
+    #   dv = Daru::Vector.new [1, 2, 3]
+    #   puts dv.summary
+    #
+    #   =
+    #     n :3
+    #     n valid:0
+    #     median: 2
+    #     mean: 2.0000
+    #     std.dev.: 1.0000
+    #     std.err.: 0.5774
+    #     skew: 0.0000
+    #     kurtosis: -2.3333
+    def summary(indent_level=0) # rubocop:disable Metrics/AbcSize, Metrics/MethodLength
```

Comment:
> These rubocop checks are related to the Assignment Branch Condition and the Method Length. I think we can safely ignore these checks.

Which are the most important checks -- much more than spacing or argument naming. No, we can't ignore them in _new_ (or rewritten) code. There was a huge effort of making the code better & conforming to rubocop. Those that left with `disables` -- were left exactly because "this method should be completely rewritten, but for now let's leave it this way".

It is absolutely not acceptable for new/refactored code.